lighthouse-core/gather/gatherers/trace-elements.js

```diff
@@ -14,6 +14,11 @@
 const Gatherer = require('./gatherer.js');
 const pageFunctions = require('../../lib/page-functions.js');
 const TraceProcessor = require('../../lib/tracehouse/trace-processor.js');
+const {
+  addRectTopAndBottom,
+  getRectOverlapArea,
```

Comment:
i prefer `RectHelpers = require(...`, so it's clear upon usage that these functions are from another file.